style.css

```diff
@@ -531,3 +562,262 @@ footer .submit:active {
   background: #2230d2;
   color: #fff;
 }
+
+@media only screen and (min-width: 768px) {
+  body {
+    width: 1440px;
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
- [ ] The website scrolls horizontally because the website's width is wider than the viewport. Please don't give static width to the content instead, you can use `%` or `vw`.